nautobot/dcim/models/device_components.py

```diff
@@ -1277,3 +1277,28 @@ def clean(self):
 
         if not (self.parent_device or self.parent_module):
             raise ValidationError("Either parent_device or parent_module must be set")
+
+
+#
+# Virtual Device Contexts
+#
+
+
+class VDC(PrimaryModel):
+    name = models.CharField(max_length=CHARFIELD_MAX_LENGTH)
+    device = models.ForeignKey("dcim.Device", on_delete=models.CASCADE, related_name="vdcs")
+    status = StatusField(blank=False, null=False)
```

Comment:
We are missing an ID field here. Should be ID  = models.PositiveSmallIntegerField